crates/cast/tests/data/keystore/my_account.json

```diff
@@ -0,0 +1,13 @@
+{
```

Comment:
Can you give me some more context on those files?

If I understand correctly, we have a keystore that stores private keys and then we have account files that hold "public" info? or did I mix up something?

Reply:
my_key.json - keystore storing private / public key
my_account.json - obtained through `starkli account oz init my_account.json --keystore my_key.json`, holds publicly available account data (the file itself can be fetched using something like `starkli account fetch <address>`). one more thing is that when undeployed, the account json file holds the salt, instead of address, which is unrecoverable after deployment as far as i know
my_account_undeployed.json - the undeployed account file

Reply:
Cool, thanks for clarification. Is `starkli account fetch <address>` necessary to get this, or can users just use them accounts without this fetch? 

To be clear - say I have an account created by starkli. Can I just point sncast to it and call it a day, or do I have to do this `fetch` thing first?

Reply:
The starkli flow is like this:
1. create the keystore (or not, if you want to use plaintext private key)
2. create the account json file (my_account_undeployed.json as an example) - it will contain the salt instead of the address
3. prefund the account and deploy it - after that the salt field will change to address field in the account json file. the file then can be removed and completely fetched again from the network

Reply:
cool, so we can just use a file directly, without fetching it (if it exists), right?

Reply:
Roger that